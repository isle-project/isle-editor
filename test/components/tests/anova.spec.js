// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Anova from '@isle-project/components/tests/anova';
import heartdisease from './../fixtures/heartdisease.json';


// TESTS //

describe( '<Anova />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <Anova
			data={heartdisease}
			variable="Cost"
			group="Drugs"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
