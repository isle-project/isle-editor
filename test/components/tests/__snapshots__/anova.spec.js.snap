// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Anova /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    anova
  </label>
  <pre
    style={
      {
        "marginTop": 10,
      }
    }
  >
    One-Way ANOVA

Null Hypothesis: All Means Equal
Alternate Hypothesis: At Least one Mean not Equal

              df    SS                 MS               F Score  P Value
Treatment     2     933297240.3279     466648620.1639   10.6822  0
Errors        785   34292496222.114    43684708.5632
  </pre>
</div>
`;
